official/projects/fffner/README.md

```diff
@@ -0,0 +1,18 @@
+# FFF-NER tf ver.
+First, we need to convert a model to our encoder structure, using `./utils/convert_checkpoint`. This will create a 
+bert-uncased tf encoder from huggingface's bert-base-uncased model.  
```

Comment:
Is converting HF checkpoint optional?
I think we already provide TF checkpoints.